src/commands/install.ts

```diff
@@ -23,21 +23,17 @@ const parseProviders = (
   if (!providers) {
     return [];
   }
+  const splitted: string[] = [];
+  providers.forEach(provider => {
+    splitted.push(...provider.split(','));
+  });
```

Comment:
There's `flatMap` that you can use like `providers.flatMap(provider => provider.split(','))` and then you can just chain the .map and .filter and become the Iterator Man

Reply:
Good point👍